src/test/java/bio/terra/service/job/JobServiceTest.java

```diff
@@ -109,6 +112,35 @@ public void tearDown() throws Exception {
     jobIds.forEach(this::deleteJob);
   }
 
+  @Test
+  public void enumerateTooLongBackFilterTest() throws Exception {
+    List<JobModel> expectedJobs = new ArrayList<>();
+    int numVisibleJobs = 3;
+    for (int i = 0; i < numVisibleJobs; i++) {
+      String jobId = runFlight(makeDescription(i), makeFlightClass(i), testUser);
+      expectedJobs.add(
+          new JobModel()
+              .id(jobId)
+              .jobStatus(JobStatusEnum.SUCCEEDED)
+              .statusCode(HttpStatus.I_AM_A_TEAPOT.value())
+              .description(makeDescription(i))
+              .className(makeFlightClass(i).getName()));
+    }
```

Comment:
You could stream here
```suggestion
    List<JobModel> expectedJobs =
        IntStream.range(0, 3)
            .mapToObj(i -> {
              String jobId = runFlight(makeDescription(i), makeFlightClass(i), testUser);
              return new JobModel()
                  .id(jobId)
                  .jobStatus(JobStatusEnum.SUCCEEDED)
                  .statusCode(HttpStatus.I_AM_A_TEAPOT.value())
                  .description(makeDescription(i))
                  .className(makeFlightClass(i).getName())
            });
```